assets/src/styles/components/_graph.scss

```diff
@@ -208,6 +208,11 @@ $provisional-color: color('accent-warm-dark');
             text-align: center;
             span {
                 padding: 0 1em 0 .5em;
+                &.show-lithology {
```

Comment:
I don't know whether it's the padding, but on my Chrome when the checkbox has the focus, the blue focus box is outside on top of the checkbox.
![screen shot 2018-12-18 at 2 57 23 pm](https://user-images.githubusercontent.com/8104736/50182546-390db500-02d5-11e9-8fa8-805cd548c5fb.png)